### ISSUE TITLE
Protected method called in 2.3.7

### ISSUE THREAD

**Issue by theodorton** [mbleigh/seed-fu#123]
We just upgraded to 2.3.7 and ran into the following issue:

```
rake aborted!
NoMethodError: protected method `postgresql_version' called for #<ActiveRecord::ConnectionAdapters::PostgreSQLAdapter:0x007fc5bd1caec8>
Did you mean?  post_sql_statements
(eval):1:in `block (2 levels) in run_file'
/Users/theodortonum/.rbenv/versions/2.3.1/bin/bundle:23:in `load'
/Users/theodortonum/.rbenv/versions/2.3.1/bin/bundle:23:in `<main>'
Tasks: TOP => db:seed_fu
(See full trace by running task with --trace)
```

Downgrading to 2.3.6 solves the problem.

We're using ActiveRecord 4.2.8.

**Comment by rymai** [comment]
We experience the same with AR 4.2.8 too.

**Comment by joshuawscott** [comment]
The `postgresql_version` method was made public in rails/AR 5.0
https://github.com/rails/rails/commit/8ad1eeafc2d4b0d96f0218b5272ea734564bd875

the `pg` gem exposes a `PG.library_version` method that returns the same thing, although it only checks the client's version.

**Comment by darix** [comment]
client version is useless. you need the server version.

**Comment by darix** [comment]
btw: shouldnt this be a bug against rails/rails instead? seed-fu could maybe query the server version via some fancy select statement. but fixing AR just like in the rails 5 branch seems easier.

or you know just port the app to rails 5 :)

**Comment by aserafin** [comment]
Should be fixed with 2.3.9 release
